### PR TITLE
:sparkles: command line options to configuration file

### PR DESCRIPTION
**Pull request by darkweaver87**
This PR adds other config options to config file.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -85,12 +120,25 @@ func parseCliArgs(flags *args) (*cobra.Command, error) {
...
 				return err
 			}
 
-			status, err := run(conf, kubeconfigArgs, *flags, printFlags)
+			mergeConfigAndFlags(conf, flags.Config, cmd)
+
+			if flags.printConfig {
+				printConfig(conf)
+				return nil
+			}
```
Comment (raffis) [inline cmd/main.go]: Not so sure about that printConfig support. This feels like it does not add real value and is more for debugging purposes. I would opt for removing this. 

(There is also a bug https://github.com/raffis/gitops-zombies/pull/56/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R245 where you can run into a nil pointer panic)

File: cmd/main.go [hunk 2]
```diff
@@ -70,7 +72,40 @@ func toExitCode(codeStr string) int {
...
+func parseCliArgs() (*cobra.Command, error) {
+	flags := args{Config: gitopszombiesv1.Config{
+		TypeMeta:         metav1.TypeMeta{},
+		ExcludeClusters:  nil,
+		ExcludeResources: nil,
+		Fail:             boolPtr(false),
+		IncludeAll:       boolPtr(false),
+		LabelSelector:    strPtr(""),
+		NoStream:         boolPtr(false),
+	}}
```
Comment (raffis) [inline cmd/main.go]: Can't we just have values in instead pointers in the v1.Config api? I reckon that is the nicer approach and you would not need to initialize the config that way.

File: pkg/apis/gitopszombies/v1/types.go [hunk 3]
```diff
@@ -9,10 +9,16 @@ import (
...
 type Config struct {
-	metav1.TypeMeta  `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+
+	ExcludeClusters  *[]string          `json:"excludeClusters,omitempty"`
 	ExcludeResources []ExcludeResources `json:"excludeResources,omitempty"`
+	Fail             *bool              `json:"fail,omitempty"`
+	IncludeAll       *bool              `json:"includeAll,omitempty"`
+	LabelSelector    *string            `json:"selector,omitempty"`
+	NoStream         *bool              `json:"noStream,omitempty"`
```
Comment (raffis) [inline pkg/apis/gitopszombies/v1/types.go]: See comment above ☝🏻

**Review by raffis — COMMENTED** [review]
Nice 🚀 , thx for the pr.

We should change the config to values and remove the print function, everything else lgtm.

**Review by raffis — COMMENTED** [review]
Looking good 👍🏻 , can you resolve the conflicts which occurred after merging your other contrib? 🙏🏻

**Comment by darkweaver87** [comment]
Just pushed :sweat_smile:

**Review by raffis — APPROVED** [review]
Thanks for the contribution ❤️

**Pull request merged**